src/Components/ContentDelivery/ContentDelivery.js

```diff
@@ -0,0 +1,27 @@
+import React, { useContext } from 'react'
+import './ContentDelivery.scss'
+import ReactPlayer from 'react-player'
+import { VideoContext } from '../../Context/VideoContext'
+
+const ContentDelivery = () => {
+  const [ videoLink ] = useContext(VideoContext)
+
+  return (
+    <section className="video-section">
+      <h2>Enjoy Your Break!</h2>
+      <div className="video-wrapper">
+        <ReactPlayer
+          className="video-player"
+          url={videoLink}
```

Comment:
It looks like this is where the url will fit in to the BE for the video!

Reply:
Yes! The url will actually be set elsewhere (in a component in progress), but then it's passed here to display!